master/custom/workers.py

```diff
@@ -147,6 +147,11 @@ def get_workers(settings):
             tags=['windows', 'win10', 'arm', 'arm32'],
             branches=['3.x'],
         ),
+        cpw(
+            name="pablogsal-arch-x86_64",
+            tags=['linux', 'unix', 'arch', 'x86-64'],
```

Comment:
Is this 32-bit or 64-bit?  If 32, the tag should be `x86`; if 64, there should also be `amd64`.

Note that I don't think we're actually using these tags anywhere yet, so it doesn't *really* matter :)

It does have implications for the builder name, though; the `x86` in that name makes it sound like 32-bit, but the `x86_64` in the worker name makes it sound like 64 bit.

Reply:
Is 64-bit (intel processor). Thanks for the heads up, I will correct the builder name soon :)

Reply:
Just to confirm, what the correct name and tags should be by our convention?

Reply:
`tags=['linux', 'unix', 'arch', 'amd64', 'x86-64'],`